src/docs/product/issues/issue-details/performance-issues/slow-db-queries/index.mdx

```diff
@@ -23,6 +23,12 @@ You can configure detector thresholds for slow DB queries issues in **Project Se
 
 ![Slow DB Query detector threshold settings](slow-db-queries-detector-settings.png)
 
+<Note>
+
+These settings can only be edited by users with the organization-level owner, manager, or team-level admin roles.
```

Comment:
```suggestion
These settings can only be edited by users with owner, manager, or team-level admin roles.
```

I think the in-app message is a bit confusing. I assume that most users don't know that we have org-level and team-level roles. Might be better to simplify a bit in both docs and the product UI?